src/components/footer/Footer.css

```diff
@@ -43,6 +43,7 @@ footer {
     width: 60px;
     height: 60px;
     padding-right: 0.4rem;
+    border-radius: 0;
```

Comment:
This is not needed if you fix in InterviewPage.css

Reply:
I removed 'border-radius' from the footer.css and set 'border-radius' of all images to 0. Now all the images would not affected by the border-radius as before.

And only apply border-radius:50% for the interview profile img to make it round 
![image](https://github.com/1thing-org/1thing.org/assets/105839955/62588e31-6049-4fa2-8c6f-b86fdb054976)
![image](https://github.com/1thing-org/1thing.org/assets/105839955/462d557f-0e61-40b6-a13c-f0ae7db58d62)